stockflux-securities-master/src/components/InputForm.js

```diff
@@ -30,75 +59,94 @@ const InputForm = ({ match }) => {
-      <form className="input-form-body" onSubmit={submitForm}>
-        <div className="input-row">
-          <label className="input-label" htmlFor="name-input">
-            Name
-          </label>
-          <input
-            className="input-form-input"
-            id="name-input"
-            value={name}
-            onChange={event => setName(event.target.value)}
-          />
-        </div>
-        <div className="input-row">
-          <label className="input-label" htmlFor="exchange-input">
-            Exchange
-          </label>
-          <input
-            className="input-form-input"
-            id="exchange-input"
-            value={exchange}
-            onChange={event => setExchange(event.target.value)}
-          />
-        </div>
-        <div className="input-row">
-          <label className="input-label" htmlFor="symbol-input">
-            Symbol
-          </label>
-          <input
-            className="input-form-input"
-            id="symbol-input"
-            value={symbol}
-            onChange={event => setSymbol(event.target.value)}
-          />
-        </div>
-        <div className="input-checkbox-container">
-          <label className="input-label" htmlFor="is-shown-toggle">
-            Is Shown
-          </label>
-          <input
-            className="input-form-toggle"
-            id="is-shown-toggle"
-            value={isShown}
-            type="checkbox"
-            onChange={() => setIsShown(!isShown)}
-          />
-          <label className="toggle-switch" htmlFor="is-shown-toggle" />
-        </div>
-        <div className="input-checkbox-container">
-          <label className="input-label" htmlFor="is-enabled-toggle">
-            Is Enabled
-          </label>
-          <input
-            className="input-form-toggle"
-            id="is-enabled-toggle"
-            value={isEnabled}
-            type="checkbox"
-            onChange={() => setIsEnabled(!isEnabled)}
-          />
-          <label className="toggle-switch" htmlFor="is-enabled-toggle" />
-        </div>
-        <div className="input-submit-button-container">
-          <div
-            className={(inProgress ? "in-progress " : "") + "input-submit-button"}
-          >
-            <button>Create</button>
+      <div className="input-form-title">
+        {match.params.securityId ? "Edit Security" : "Create a Security"}
+      </div>
+      {isLoading ? (
+        <Components.LargeSpinner />
+      ) : errorMessage ? (
+        <ErrorMessage errorMessage={errorMessage} />
+      ) : (
+        <form className="input-form-body" onSubmit={submitForm}>
+          <div className="input-row">
+            <label className="input-label" htmlFor="exchange-input">
+              Exchange
+            </label>
+            <input
+              className={`input-form-input${
+                match.params.securityId ? "-read-only" : ""
```

Comment:
See my comment above when I suggested having the `input-form-input` class always present on inputs and adding the `*-read-only` modifier class when appropriate.

Reply:
yep done